runner/src/server/plugins/blankie.ts

```diff
@@ -11,15 +11,20 @@ export const configureBlankiePlugin = (
   return {
     plugin: Blankie,
     options: {
-      fontSrc: ["self", "data:"],
-      scriptSrc: (() =>
-        ["self", "unsafe-inline"].concat(
+      connectSrc: (() =>
+        ['self'].concat(
           config.matomoUrl ? [config.matomoUrl] : []
         ))(),
-      styleSrc: ["self", "unsafe-inline"],
+      defaultSrc: ['self'],
+      fontSrc: ['self', 'data:'],
+      scriptSrc: (() =>
+        ['self', 'unsafe-inline'].concat(
+          config.matomoUrl ? [config.matomoUrl + '/piwik/piwik.js'] : []
```

Comment:
it would be worth using nodeJS's URL 
`const myURL = new URL('/foo', 'config.matomoUrl');` URL will deal with trailing/leading or no trailing/leading slashes safely